app/controllers/ctc/questions/email_verification_controller.rb

```diff
@@ -2,13 +2,13 @@ module Ctc
   module Questions
     class EmailVerificationController < QuestionsController
       include AnonymousIntakeConcern
+      before_action :redirect_if_duplicate_ctc_client
       before_action :send_verification_code, only: [:edit]
 
       layout "intake"
 
       # if the client already has a valid intake with same info, don't have them verify again.
       def self.show?(intake)
-        return false if ClientLoginService.has_ctc_duplicate?(intake)
         return false if intake.sms_phone_number_verified_at? # only require one verified contact type
```

Comment:
yeah, this area of the code is like... kind of brittle because of the fact that its really difficult to ensure that you re-verify after you make changes, etc. I actually think that maybe you're right though and that this page shouldn't show if you've already verified because you don't want to be able to go "back" to this page. I'll check it out.